docs/_includes/attr-doc.adoc

```diff
@@ -94,11 +94,17 @@ The header substitution group replaces <<user-manual#special-characters,special
 The header substitution group is applied to metadata lines (author and revision information) in the document header.
 This group is also applied to the values of attribute entries, regardless of whether those entries are defined in the header or elsewhere in the document.
 
-If you require different substitutions to be applied to an attribute's value, you can use the <<user-manual#pass-macros,inline pass macro>>.
-In order for the inline macro to be processed, it must completely enclose the attribute value.
-It's used to apply alternate substitutions to the enclosed text at the time the attribute is defined.
+If you require different substitutions to be applied to an attribute's value, there are two approaches from which to choose.
```

Comment:
I think saying "two approaches" here is confusing. In fact, I don't think there are two approaches. This is the sole way to change the substitutions that are applied.

Reply:
Well, what I was trying to refer to there (though apparently unclearly) were the following:

1) changing the substitutions at the point of attribute-**entry** (via a surrounding inline pass macro)
2) changing the substitutions at the point of attribute-**reference** (via changing the order of subs)
The latter one is mentioned in the text, of course, just past the point where I've edited. 

I take it that I was not clear that this second one was the other "approach" I meant. I'm open to any and every suggestion of ways I could have said it clearer. 
(I don't know if making it sound like these are exclusive alternatives (when in fact you COULD apply both, though it would be odd to WANT to do that) is misleading or not...).

Reply:
But I'm starting to see your point now: this whole subsection is entitled, "Substitutions in an attribute entry", so we don't really need to slap all that verbiage in there just to clarify that.

I guess that why I felt this PR would be helpful, was that it clarified for me (based on your explanation) that the pass macro was, here, basically a way to override (via "pre-processing", using your metaphor) the normal set of substitutions that would apply. 

Let me try to clean up that first paragraph I edited -- get it back to what it was, basically, before my last commit. But also, I will suggest one additional word (only) for clarification, in that part I over-edited:

"If you require different substitutions to be applied to an attribute **entry**’s value, ..."